GoRogue/MapGeneration/Region.Properties.cs

```diff
@@ -141,7 +137,7 @@ public int Bottom
         /// <summary>
         /// All points in this region
         /// </summary>
-        public IEnumerable<Point> Points => OuterPoints.Concat(InnerPoints).Concat(Connections);
+        public Area Points => (Area) _outerPoints.Positions.Concat(_innerPoints.Positions).Concat(_connections.Positions);
```

Comment:
Don't think there exists an implicit/explicit conversion to `Area` from `IEnumerable`: this will probably crash.  I believe this property needs to be `IEnumerable<Point>` in return value since it's just a combination of existing areas.